introduction.md

```diff
@@ -0,0 +1 @@
+# Hey team! I am the new dojo guy. it is nice to meet you all.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.